.github/workflows/check_scala3_nightly.yml

```diff
@@ -10,7 +10,9 @@ jobs:
       - uses: olafurpg/setup-scala@v13
       - name: "Query maven-central"
         run: |
-          TODAY_NIGHTLY=$(./coursier complete org.scala-lang:scala3-compiler_3: | grep $(date '+%Y%m%d'))
+          curl -fLo cs https://git.io/coursier-cli-"$(uname | tr LD ld)"
```

Comment:
I thought that `coursier` comes `olafurpg/setup-scala@v13`. If not, could you remove it (`- uses: olafurpg/setup-scala@v13`)?

Reply:
Actually, instead of using setup-scala here and then manually installing coursier you could also just consider using https://github.com/coursier/setup-action, which will still ensure you have everything you need to use coursier, without any extra work.

Reply:
switched it on `coursier/setup-action`